proto/storage/node.proto

```diff
@@ -49,6 +49,7 @@ message Node {
     // Time we received an update from Kubernetes.
     google.protobuf.Timestamp k8s_updated  = 26 [(gogoproto.moretags) = 'sensorhash:"ignore"'];
 
+    NodeInventory node_inventory           = 28 [(gogoproto.moretags) = 'policy:"Node Inventory" sql:"-"'];
```

Comment:
Add a comment that this will not be stored.
Check that this is not stored in the Node table.